### ISSUE TITLE
GIVpower Boosting | Feed back from a confused user on the first boost, need to remove the slider

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-dapps-v2#1934]
<img width="526" alt="Screen Shot 2022-12-13 at 12 35 04 PM" src="https://user-images.githubusercontent.com/14945613/207404221-fc20c770-8d31-44d6-a884-0f74e6392977.png">

We should remove the slider for the first boost

**Comment by laurenluz** [comment]
yeah ok, let's just hide the slider on the 1st boost @MohammadPCh - I think that's easy enough and solves the problem.

**Comment by MohammadPCh** [comment]
@laurenluz @maryjaf could you please check this on the preview link?
https://giveth-dapps-v2-git-add-boost-button-to-project-card-givethio.vercel.app/

**Comment by maryjaf** [comment]
> @laurenluz @maryjaf could you please check this on the preview link? https://giveth-dapps-v2-git-add-boost-button-to-project-card-givethio.vercel.app/
Thanks @MohammadPCh 
I tested and when I tap on this icon the project page is open and the modal of boosting is shown
<img width="1380" alt="image" src="https://user-images.githubusercontent.com/111529185/207587132-08f95673-37db-4596-b315-71bf281d30dd.png">

**Comment by maryjaf** [comment]
> @laurenluz @maryjaf could you please check this on the preview link? https://giveth-dapps-v2-git-add-boost-button-to-project-card-givethio.vercel.app/
Thanks @MohammadPCh Now for the first project this modal is shown
<img width="485" alt="image" src="https://user-images.githubusercontent.com/111529185/207606644-3da88f7f-ad4c-4753-98b7-cae8366f92f2.png">

**Comment by markoprljic** [comment]
Yeah this was confusing me as well. Glad to see you solved it. Also, Manage my GIVpower is redundant because first time opening that page it will be empty. I would remove that link as well.

**Comment by laurenluz** [comment]
@MohammadPCh can you do that too?

- [ ] So remove the sentence "You can review and manage your GIVpower allocations in My GIVpower"
- [ ] And remove the link under the button that says "Manage your GIVpower"

**Comment by MohammadPCh** [comment]
@laurenluz  for sure :)
